### PR TITLE
Library angular

### PR DESCRIPTION
**Pull request by Conroyce**
create student view for library, create core view for library

### CODE REVIEW

**Comment by krish-andres** [comment]
Tested and approved!

File: assets/js/directives/admin_library_directive.js [hunk 1]
```diff
@@ -1,4 +1,4 @@
-app.directive("libraryDirective", ["Library", "$location", "$route", "Module",function(Library, $location, $route, Module) {
+app.directive("adminLibraryDirective", ["Library", "$route", "Module",function(Library, $route, Module) {
```
Comment (mmomen) [inline assets/js/directives/admin_library_directive.js]: Should have a space before the function

File: assets/js/templates/core_library_template.html [hunk 2]
```diff
@@ -0,0 +1,28 @@
+<div class="container">
+  <div class="row">
+    <h2 class="col-md-12">Library View - Core</h2>
+  </div>
+
+  <div class="row">
+    <div class="col-md-4">
+      <input type="text" placeholder="New Module" class="form-control" ng-model="name">
```
Comment (mmomen) [inline assets/js/templates/core_library_template.html]: I believe it's best practice to have the value in ng-model be an object that is intialized in the controller and you assign it a property here. ex. "ng-model=newModule.name" and in the controller "$scope.newModule = {}"

**Comment by mmomen** [comment]
Made 2 minor comments, otherwise tested and it works!

**Comment by delasky** [comment]
Connor you should squash and then I will merge

**Pull request merged**